x-pack/test/reporting_api_integration/reporting_without_security/job_apis_csv.ts

```diff
@@ -70,9 +70,11 @@ export default function ({ getService }: FtrProviderContext) {
           "created_by": false,
           "jobtype": "csv_searchsource",
           "meta": Object {
+            "isDeprecated": false,
```

Comment:
I'm not sure how this snapshot changed. I think the previous code suppresses this field when the value is not `true`.